src/Symfony/Bundle/WebProfilerBundle/composer.json

```diff
@@ -27,6 +27,9 @@
         "symfony/dependency-injection": "~2.8|~3.0",
         "symfony/stopwatch": "~2.8|~3.0"
     },
+    "conflict": {
+        "symfony/http-kernel": "<3.1"
```

Comment:
You should then update the require section instead. And we should probably add a test if possible.